src/webgpu/api/operation/storage_texture/read_only.spec.ts

```diff
@@ -414,9 +414,8 @@ g.test('basic')
       .combine('depthOrArrayLayers', [1, 2] as const)
   )
   .beforeAllSubcases(t => {
-    if (t.params.format === 'bgra8unorm') {
-      t.selectDeviceOrSkipTestCase('bgra8unorm-storage');
-    }
```

Comment:
`t.selectDeviceOrSkipTestCase('bgra8unorm-storage')` is still needed because `bgra8unorm` can be used for storage textures only when the extension `bgra8unorm-storage` is supported.